dev/ProgressBar/TestUI/ProgressBarPage.xaml.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Markup;
+using Windows.UI;
+using System.Windows.Input;
+
+using ProgressBar = Microsoft.UI.Xaml.Controls.ProgressBar;
+//using Windows.UI.Xaml.Tests.MUXControls.ApiTests.RepeaterTests.Common;
+using Windows.UI.Xaml.Data;
+using System.Numerics;
+using Windows.UI.Xaml.Shapes;
+using System.Security.Cryptography;
```

Comment:
this seems weird. what are you using cryptography for? :smile:

Reply:
hmm.. it seemed to have gotten added in automatically again. Let me remove it